pkg/cloudprovider/providers/azure/azure_blobDiskController.go

```diff
@@ -106,7 +106,7 @@ func (c *BlobDiskController) CreateVolume(name, storageAccount string, storageAc
 		return "", "", 0, err
 	}
 
```

Comment:
useHTTPSForBlobBasedDisk?

Reply:
thanks for the check, I should use useHTTPSForBlobBasedDisk